spidermon/contrib/scrapy/monitors.py

```diff
@@ -101,12 +101,12 @@ class UnwantedHTTPCodesMonitor(BaseScrapyMonitor):
             code: 10
             for code in [400, 407, 429, 500, 502, 503, 504, 523, 540, 541]}
 
-    **WARNING**: You can have this settings by spider like::
+    **WARNING**: You CAN NOT have this settings by spider like::
```

Comment:
Both cannot and can not are acceptable spellings, but the first is much more usual. You would use can not when the ‘not’ forms part of another construction such as ‘not only’. For example:

> These green industries can not only create more jobs, but also promote sustainable development of the land.

Reference: https://en.oxforddictionaries.com/usage/cannot-or-can-not